gremlin-groovy/src/test/java/org/apache/tinkerpop/gremlin/groovy/jsr223/GroovyTranslatorProvider.java

```diff
@@ -215,6 +215,10 @@
         test = "org.apache.tinkerpop.gremlin.process.traversal.step.map.WriteTest",
         method = "*",
         reason = "read and write tests don't translate locally well because of calling iterate() inside read()/write() add a none()")
+@Graph.OptOut(
+        test = "org.apache.tinkerpop.gremlin.process.traversal.step.OrderabilityTest",
+        method = "g_inject_order_with_unknown_type",
+        reason = "Cannot round trip a generic Java Object")
```

Comment:
Maybe update the reason here to say something about how this doesn't work for remote. To a layman code reader, this leaves a cliffhanger about why cannot be roundtrip generic objects here.